From 3a96993a5dd17061b4c95dc67777ddd0d807080a Mon Sep 17 00:00:00 2001
From: Raphael Mounier <mounierr07@gmail.com>
Date: Sun, 14 Dec 2025 09:52:23 +0100
Subject: [PATCH] Disable vndklite handling

Don't check ro.vndk.lite property and return always false
---
 modules/environment.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/modules/environment.cc b/modules/environment.cc
index 559a5ec..5e455f9 100644
--- a/modules/environment.cc
+++ b/modules/environment.cc
@@ -25,7 +25,7 @@ namespace linkerconfig {
 namespace modules {
 
 bool IsVndkLiteDevice() {
-  return Variables::GetValue("ro.vndk.lite").value_or("") == "true";
+  return false;
 }
 
 bool IsVndkInSystemNamespace() {
-- 
2.25.1

